tabcmd/commands/user/user_command.py

```diff
@@ -65,14 +69,20 @@ def validate_file_for_import(csv_file, logger, detailed=False):
     @staticmethod
     def validate_username(username):
         if username is None or username == "" or username.strip(" ") == "":
+            logger.debug("Username must be specified in csv file")
             raise AttributeError("Username must be specified in csv file")
         if username.find(" ") >= 0:
+            logger.debug("Username cannot contain spaces")
             raise AttributeError("Username cannot contain spaces")
         at_symbol = username.find("@")
 
         if at_symbol >= 0:
-            username = username[:at_symbol] + "X" + username[at_symbol + 1 :]
+            username = username[:at_symbol] + "X" + username[at_symbol + 1:]
             if username.find("@") >= 0:
+                logger.debug(
```

Comment:
do we need the debug statement? Won't the exception just get logged anyway?

Reply:
Thanks, I just checked with a sample and you are absolutely correct. The errors raised were indeed logged to the log file. We don't need these extra debug statements